.github/workflows/test-windows.yml

```diff
@@ -143,14 +171,12 @@ jobs:
 
     - name: Build wheel
       id: wheel
-      if: "github.event_name == 'push'"
```

Comment:
This change means that wheels will be built on PRs. What's your thinking behind this?

Reply:
I just figured there is no reason not to. Before it was split based on whether libimagequant was being built, with this PR the build is the same (otherwise it would never cache due to security reasons). If you think the artifact size is a concern, I can remove it.

Reply:
This change is now reverted.